src/MarginTrading.DataReader/Modules/DataReaderRepositoriesModule.cs

```diff
@@ -9,72 +9,76 @@
-			).SingleInstance();
-
-			builder.Register<IMatchingEngineRoutesRepository>(ctx =>
-				AzureRepoFactories.MarginTrading.CreateMatchingEngineRoutesRepository(_settings.Db.MarginTradingConnString, _log)
-			).SingleInstance();
-
-			builder.Register<IMarginTradingConditionRepository>(ctx =>
-				AzureRepoFactories.MarginTrading.CreateTradingConditionsRepository(_settings.Db.MarginTradingConnString, _log)
-			).SingleInstance();
-
-			builder.Register<IMarginTradingAccountGroupRepository>(ctx =>
-				AzureRepoFactories.MarginTrading.CreateAccountGroupRepository(_settings.Db.MarginTradingConnString, _log)
-			).SingleInstance();
-
-			builder.Register<IAccountAssetPairsRepository>(ctx =>
-				AzureRepoFactories.MarginTrading.CreateAccountAssetsRepository(_settings.Db.MarginTradingConnString, _log)
-			).SingleInstance();
-
-			builder.Register<IAssetPairsRepository>(ctx =>
-				AzureRepoFactories.MarginTrading.CreateAssetsRepository(_settings.Db.DictsConnString, _log)
-			).SingleInstance();
-
-			builder.Register<IMarginTradingBlobRepository>(ctx =>
-				AzureRepoFactories.MarginTrading.CreateBlobRepository(_settings.Db.StateConnString)
-			).SingleInstance();
-
-			builder.Register<IAppGlobalSettingsRepositry>(ctx =>
-				AzureRepoFactories.Settings.CreateAppGlobalSettingsRepository(_settings.Db.ClientPersonalInfoConnString, _log)
-			).SingleInstance();
-		}
-	}
+    public class DataReaderRepositoriesModule : Module
+    {
+        private readonly MarginSettings _settings;
+        private readonly ILog _log;
+
+        public DataReaderRepositoriesModule(MarginSettings settings, ILog log)
+        {
+            _settings = settings;
+            _log = log;
+        }
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder.RegisterInstance(_log)
+                .As<ILog>()
+                .SingleInstance();
+
+            builder.Register<IMarginTradingOperationsLogRepository>(ctx =>
+                new MarginTradingOperationsLogRepository(
+                    AzureTableStorage<OperationLogEntity>.Create(() => _settings.Db.LogsConnString,
+                        "MarginTradingDataReaderOperationsLog", _log))
+            ).SingleInstance();
+
+            builder.Register<IClientSettingsRepository>(ctx =>
+                AzureRepoFactories.Clients.CreateTraderSettingsRepository(_settings.Db.ClientPersonalInfoConnString, _log)
+            ).SingleInstance();
+
+            builder.Register<IMarginTradingAccountsRepository>(ctx =>
+                AzureRepoFactories.MarginTrading.CreateAccountsRepository(_settings.Db.MarginTradingConnString, _log)
+            ).SingleInstance();
+
+            builder.Register<IMarginTradingAccountStatsRepository>(ctx =>
+                AzureRepoFactories.MarginTrading.CreateAccountStatsRepository(_settings.Db.MarginTradingConnString, _log)
+            ).SingleInstance();
```

Comment:
Better move to HistoryConnString or StateConnString or create new ReportsConnString